components/ContactUs/ContactUs.jsx

```diff
@@ -0,0 +1,42 @@
+import labels from '@/constants/labels.en'
+import Image from 'next/image'
+import styles from '@/styles/ContactUs.module.css'
+
+const ContactUs = () => {
+      return (
+        <div>
+          <div>
+            <h3 className={styles.title} >{labels.contactUs.title}</h3>
+          </div>
+
+          <div className='padding'>
+            <h4 className={styles.subtitle }>{labels.chat.chat}</h4>
+            <p className='b2 body2'>{labels.chat.descriptionChat}</p>
+          </div>
+
+            <div >
+             <address >
+              <Image className = {styles.mailimg } width="20" height="16" src="/vector.svg" alt="mail" />
+              <a className={styles.mail + ' body3'+' b3'} href={labels.email.malito}>
```

Comment:
you are concatenating body3 and b3 styles. There's no style named body3b3

Reply:
done